app/src/main/res/values/strings.xml

```diff
@@ -4,136 +4,137 @@
     <string name="channel_subscriber_service_name">Subscription Service</string>
     <string name="channel_subscriber_notification_title">Listening for incoming notifications</string>
-    <string name="channel_subscriber_notification_instant_text">You are subscribed to instant delivery topics</string>
-    <string name="channel_subscriber_notification_instant_text_one">You are subscribed to one instant delivery topic</string>
-    <string name="channel_subscriber_notification_instant_text_two">You are subscribed to two instant delivery topics</string>
-    <string name="channel_subscriber_notification_instant_text_three">You are subscribed to three instant delivery topics</string>
-    <string name="channel_subscriber_notification_instant_text_four">You are subscribed to four instant delivery topics</string>
-    <string name="channel_subscriber_notification_instant_text_more">You are subscribed to %1$d instant delivery topics</string>
-    <string name="channel_subscriber_notification_noinstant_text">You are subscribed to topics</string>
-    <string name="channel_subscriber_notification_noinstant_text_one">You are subscribed to one topic</string>
-    <string name="channel_subscriber_notification_noinstant_text_two">You are subscribed to two topics</string>
-    <string name="channel_subscriber_notification_noinstant_text_three">You are subscribed to three topics</string>
-    <string name="channel_subscriber_notification_noinstant_text_four">You are subscribed to four topics</string>
-    <string name="channel_subscriber_notification_noinstant_text_more">You are subscribed to %1$d topics</string>
+    <string name="channel_subscriber_notification_instant_text">Subscribing to instant delivery topics</string>
+    <string name="channel_subscriber_notification_instant_text_one">Subscribing to one instant delivery topic</string>
+    <string name="channel_subscriber_notification_instant_text_two">Subscribing to two instant delivery topics</string>
+    <string name="channel_subscriber_notification_instant_text_three">Subscribing to three instant delivery topics</string>
+    <string name="channel_subscriber_notification_instant_text_four">Subscribing to four instant delivery topics</string>
+    <string name="channel_subscriber_notification_instant_text_more">Subscribing to %1$d instant delivery topics</string>
+    <string name="channel_subscriber_notification_noinstant_text">Subscribing to topics</string>
+    <string name="channel_subscriber_notification_noinstant_text_one">Subscribing to one topic</string>
+    <string name="channel_subscriber_notification_noinstant_text_two">Subscribing to two topics</string>
+    <string name="channel_subscriber_notification_noinstant_text_three">Subscribing to three topics</string>
+    <string name="channel_subscriber_notification_noinstant_text_four">Subscribing to four topics</string>
+    <string name="channel_subscriber_notification_noinstant_text_more">Subscribing to %1$d topics</string>
```

Comment:
```suggestion
    <string name="channel_subscriber_notification_noinstant_text_more">%1$d topic subscriptions</string>
```

Reply:
I was thinking of examples such as this, derived from lucrative testing budgets
![bilde](https://user-images.githubusercontent.com/13802408/160273175-79715a43-03be-4e6b-8b83-c59a43ec0997.png)

Reply:
I think I'm good with Subscribed. ;-)